scripts/image_embeddings.py

```diff
@@ -48,6 +48,10 @@ def store_embeddings(row):
             logging.info(err)
             logging.info(row.Filename)
             return
+        except OSError as err:
+            logging.info(err)
+            logging.info(row.Filename)
+            return
```

Comment:
This was the simplest way to get the script working again with minimal effort, but it's very bad. We could check that the `OSError` is really the one we want to skip over with some string matching, but it's a bit tedious.

Presumably there is a way to do some kind of filtering from the intake `yaml`, but I don't want to dig into this right now, especially if we are not entirely sold on `intake` (at least version 0.7) yet.

Reply:
Aye, got to prioritise and my inclination at this point is to back away from `intake` as a component choice - it's here because `scivision` was tightly integrated with it, and there are other choices either more standards oriented (STAC) or research ecosystem focused (Ro-Crate) https://www.researchobject.org/ro-crate/about_ro_crate